src/OSPSuite.Core/Extensions/CurveExtensions.cs

```diff
@@ -37,5 +40,37 @@ public static void UpdateMarkerCurve(this Curve curve, string curveName)
          curve.LineThickness = 2;
          curve.Name = curveName;
       }
+
+      public static void UpdateIdentityCurve(this Curve curve, string curveName)
+      {
+         curve.VisibleInLegend = false;
+         curve.LineStyle = LineStyles.Solid;
+         curve.Color = Color.Black;
+         curve.Name = curveName;
+         curve.LineThickness = 2;
+         curve.Name = curveName;
+      }
+
+      public static void UpdateDeviationCurve(this Curve curve, string curveName, int curveNumber)
```

Comment:
curveNumber? Can we create an enum for this (UPPER, MIDDLE, LOWER instead of 0 1 etc..)

Reply:
this is not upper, lower aso, this is literally the count of deviation lines in the chart (possibly CurveCount would be a better naming now that I think of it), since we are setting a different LineStyle to each new deviation line with a rotation of 3

Reply:
renamed to curve count